packages/manager/apps/web/client/app/domain/list/template.html

```diff
@@ -308,7 +308,7 @@
             <oui-action-menu-item
                 data-on-click="$ctrl.goToResiliateCancelation($row)"
                 aria-label="{{:: 'domains_action_per_line_resiliation_cancellation' | translate }}"
-                data-ng-if="$row.renewalState === $ctrl.DOMAIN_RENEWABLE_STATE.CANCELLATION_REQUESTED"
+                data-ng-if="$row.contactAdmin.id === $ctrl.user.auth.account && $row.renewalState === $ctrl.DOMAIN_RENEWABLE_STATE.CANCELLATION_REQUESTED"
```

Comment:
Is it OK for CA region as well ?

Reply:
yep checked it and CA region returned Id with nic account too (not mail)